File: 0088-merge-sorted-array/0088-merge-sorted-array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        for(int i = 0; i < n; i++){
            std::replace(nums1.begin() + m + i, nums1.begin() + m + i + 1, 0, nums2[i]);
        }
        sort(nums1.begin(), nums1.end());
    }
};

File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        merged = []
        c1 = 0
        c2 = 0

        while c1 < m and c2 < n:
            if nums1[c1] <= nums2[c2]:
                merged.append(nums1[c1])
                c1 += 1
            else:
                merged.append(nums2[c2])
                c2 += 1
        
        while c1 < m:
            merged.append(nums1[c1])
            c1 += 1
        
        while c2 < n:
            merged.append(nums2[c2])
            c2 += 1

        for i in range(len(merged)):
            nums1[i] = merged[i]